resources/views/layouts/nav.blade.php

```diff
@@ -21,6 +21,7 @@
             <ul class="nav navbar-nav">
                 <li><a href="{{ url('/home') }}">Home</a></li>
                 <li><a href="/jobs">Jobs</a></li>
+                <li><a href="/blog/about">About</a></li>
```

Comment:
sure, but before we name the route should we leave it as blog? docs makes more sense in the context of the repo but blog makes sense for the site. Currently I've got the folder called docs and the route called blog, which isn't a great naming system